lib/AppInfo/Application.php

```diff
@@ -129,25 +129,19 @@ public function register(IRegistrationContext $context): void {
 
 
 	/**
-	 * @param IBootContext $context
-	 *
 	 * @throws Throwable
 	 */
 	public function boot(IBootContext $context): void {
-		$serverContainer = $context->getServerContainer();
-
 		$context->injectFn(function (IShareManager $shareManager) {
 			$shareManager->registerShareProvider(ShareByCircleProvider::class);
 		});
 
-		$this->configService = $context->getAppContainer()
-			->get(ConfigService::class);
-
 		$context->injectFn(Closure::fromCallable([$this, 'registerMountProvider']));
 	}
 
 
-	public function registerMountProvider(IServerContainer $container) {
+	public function registerMountProvider(ContainerInterface $container) {
+		$this->configService = $container->get(ConfigService::class);
 		if (!$this->configService->isGSAvailable()) {
```

Comment:
```suggestion
		$configService = $container->get(ConfigService::class);
		if (!$configService->isGSAvailable()) {
```
It’s only used here I suppose?
It should be a local var then.